### ISSUE TITLE
Replace .Creators with .Contributors for Various Roles, i18n, and Recursive Rendering Caching Feature Supports

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#94]
# Description
```
Please provide a short description of what feature you're looking forward to
enhance below. Please include the story behind your idea as well to give a
better visualization of your idea.
```
The existing `.Creators` are overly limited to single language, was not cached (process for every sites recursively), and unable to set multiple roles that are required for LDJSON rendering. Transformation to `.Contributors` effort is required.

# Expected Behavior
```
Please specify the expected behavior of your requested enhancement. Some great
and helpful pointers are your expected interface (e.g. command patterns,
simple sketches of the user interface, etc).
```
`.Contributors` is capable if facilitating i18n support, utilize rendering caches, and allows multiple roles designation.

# Current Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
`.Creators` are not capable if providing multiple languages, being rendering on Page and each metadata every single time, and not able to designate multiple roles.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Issue related to #88  and #59 .

**Comment by hollowaykeanho** [comment]
Implemented in 14a3198961ac6893d1855bec760b2c2e42d1354b

**Comment by hollowaykeanho** [comment]
Released in version `v.1.2.0`.
